sg_otio/sg_cut_track_writer.py

```diff
@@ -947,9 +949,12 @@ def _get_shot_payload(self, clip_group, sg_project, sg_linked_entity, sg_user=No
                 # event log entry
                 if event_log:
                     reinstate_status = event_log["meta"]["old_value"]
-            elif reinstate_status_pref:
+                elif shot_reinstate_status_default:
+                    # Use the default status if we didn't find a previous status
+                    reinstate_status = shot_reinstate_status_default
+            elif shot_reinstate_status:
```

Comment:
Could be wrong, but it seems that with your if/elif structure we can set the status to `None`. Is that something that we want to do?

Reply:
it's not something that we were doing previously, no. I switched back to the original logic which seems better?